Extension/src/Debugger/attachToProcess.ts

```diff
@@ -122,12 +123,14 @@ export class RemoteAttachPicker {
 
         // Must use single quotes around the whole command and double quotes for the argument to `sh -c` because Linux evaluates $() inside of double quotes.
         // Having double quotes for the outerQuote will have $(uname) replaced before it is sent to the remote machine.
+        // Also use a full path on Linux, so that we can use transports that need a full patch such as 'machinectl' to connect to nspawn containers.
```

Comment:
Is this Linux or all of Unix? This codepath will also affect macOS.

Reply:
I don't run OSX, but /bin/sh is a thing on all unix derivatives

Reply:
I've changed to match exactly on "linux", just in case, as machinectl and nspawn run on linux only anyway